### PR TITLE
Fix style for hashes

### PR DESCRIPTION
**Pull request by AZero13**
Make the style for hashes consistent.

### CODE REVIEW

File: app/views/auth/passwords/edit.html.haml [hunk 1]
```diff
@@ -8,9 +8,9 @@
...
-      = f.input :password, wrapper: :with_label, autofocus: true, label: t('simple_form.labels.defaults.new_password'), input_html: { 'aria-label' => t('simple_form.labels.defaults.new_password'), :autocomplete => 'new-password', :minlength => User.password_length.first, :maxlength => User.password_length.last }, required: true
+      = f.input :password, wrapper: :with_label, autofocus: true, label: t('simple_form.labels.defaults.new_password'), input_html: { 'aria-label' => t('simple_form.labels.defaults.new_password'), autocomplete: 'new-password', minlength: User.password_length.first, maxlength: User.password_length.last }, required: true
     .fields-group
-      = f.input :password_confirmation, wrapper: :with_label, label: t('simple_form.labels.defaults.confirm_new_password'), input_html: { 'aria-label' => t('simple_form.labels.defaults.confirm_new_password'), :autocomplete => 'new-password' }, required: true
+      = f.input :password_confirmation, wrapper: :with_label, label: t('simple_form.labels.defaults.confirm_new_password'), input_html: { 'aria-label' => t('simple_form.labels.defaults.confirm_new_password'), autocomplete: 'new-password' }, required: true
```
Comment (larouxn) [inline app/views/auth/passwords/edit.html.haml:13]: We should probably update `'aria-label' =>` to `'aria-label':` for consistency if we want to enforce style.

**Review by larouxn — COMMENTED** [review]
Ideally We can fix these inconsistencies and prevent any future inconsistency with the [Style/HashSyntax RuboCop rule](https://docs.rubocop.org/rubocop/cops_style.html#stylehashsyntax). Looks like the default value for that rule is to enforce `symbol: "value"` style but requires additional configuration to prevent mixed key value style.

We can try adding the following RuboCop configuration to enforce a singular hash style.

```yml
Style/HashSyntax:
  Enabled: true
  EnforcedStyle: ruby19_no_mixed_keys
```

Can add it right after the follow `Style/Hash` configuration options.

https://github.com/mastodon/mastodon/blob/290d78cea4850982a2843dc1a2954f0d66fe58d8/.rubocop.yml#L231-L244

Tested locally:

image

(please disregard most of the RuboCop infractions, they're likely due to my local Ruby 3.1.2)

**Comment by AtariDreams** [comment]
Fixed!

**Review by larouxn — APPROVED** [review]
Automatically enforced and consistent. Makes sense to me. 👍

**Comment by AtariDreams** [comment]
Any update on this?

**Comment by AtariDreams** [comment]
Great!

**Comment by larouxn** [comment]
We should probably wait to ship this _after_ the RuboCop revamp PR to make sure everything is working properly.
- https://github.com/mastodon/mastodon/pull/20574

**Review by ykzts — APPROVED** [review]

**Pull request merged**